README.md

```diff
@@ -24,15 +23,12 @@ Advanced features of Taichi include [spatially sparse computing](https://taichi.
 
 <a href="https://github.com/taichi-dev/taichi/blob/master/examples/mpm_lagrangian_forces.py"><img src="https://raw.githubusercontent.com/yuanming-hu/public_files/master/graphics/taichi/lagrangian.gif" height="192px"></a> <a href="https://github.com/taichi-dev/taichi/blob/master/examples/pbf2d.py"><img src="https://raw.githubusercontent.com/yuanming-hu/public_files/master/graphics/taichi/pbf.gif" height="192px"></a> <a href="https://github.com/taichi-dev/taichi/blob/master/examples/game_of_life.py"><img src="https://raw.githubusercontent.com/yuanming-hu/public_files/master/graphics/taichi/game_of_life.gif" height="192px"></a> <a href="https://github.com/taichi-dev/taichi/blob/master/examples/euler.py"><img src="https://raw.githubusercontent.com/yuanming-hu/public_files/master/graphics/taichi/euler.gif" height="192px"></a>
 
-
-## Installation [![Downloads](https://pepy.tech/badge/taichi-nightly)](https://pepy.tech/project/taichi-nightly)
+## Installation [![Downloads](https://img.shields.io/pypi/dw/taichi)](https://pepy.tech/project/taichi)
```

Comment:
I removed the `taichi-nightly` download since you said you'll replace it with `taichi` whn we catch up, but we have no worry about this if it's counted `per-week`.

Reply:
Thanks!